Cargo.toml

```diff
@@ -50,8 +50,8 @@ rand                  = "^0.8"
 regex                 = { version = "1.5.5", optional = true, default_features = false, features = ["std"] }
 reqwest               = { version = "0.11", features = ["json"] }
 rpassword             = { version = "^5.0", optional = true }
-rpki                  = { version = "0.13.2", features = [ "compat", "repository", "rrdp", "serde" ] }
-# rpki                  = { version = "0.13.2-dev", git = "https://github.com/NLnetLabs/rpki-rs/", branch = "compat-for-prefix-lens", features = [ "compat", "repository", "rrdp", "serde" ] }
+# rpki                  = { version = "0.13.2", features = [ "compat", "repository", "rrdp", "serde" ] }
+rpki                  = { version = "0.14.3-dev", git = "https://github.com/NLnetLabs/rpki-rs/", branch = "support-rfc6492", features = [ "ca", "repository", "rrdp" ] }
```

Comment:
The "ca" feature of "rpki-rs" seems to include the "repository" feature making it redundant here, right?

Reply:
done